test/fixtures/simple-app/mixins/example.js

```diff
@@ -0,0 +1,5 @@
+module.exports = function(Model, options) {
+
+  Model.exampleMixin = true;
```

Comment:
Either assert in a unit-test that the variable was set by the mixin, or else remove this line. The same comment applies to the other mixin scripts too.